pkg/scheduler/metrics/metrics.go

```diff
@@ -159,8 +159,8 @@ var (
 			Subsystem: SchedulerSubsystem,
 			Name:      "pod_scheduling_duration_seconds",
 			Help:      "E2e latency for a pod being scheduled which may include multiple scheduling attempts.",
-			// Start with 1ms with the last bucket being [~16s, Inf)
-			Buckets:        metrics.ExponentialBuckets(0.001, 2, 15),
+			// Start with 1.875s with the last bucket being [64m, Inf)
+			Buckets:        metrics.ExponentialBuckets(1.875, 2, 12),
```

Comment:
Do we have some DP proving `1.875s` is a good starting point?

Reply:
I think the current one is certainly too small and not useful.

I expect that pods will take in the range of few seconds if scheduled on the first attempt, and that will depend on the queue length. If scheduling the pod will require multiple attempts, then I expect this to be in the tens of seconds to minutes.

Reply:
we could start with something smaller, but I wanted to align with minutes at the end.

Reply:
how about 0.1 second (100ms)?

Reply:
SG.